latest/index.bs

```diff
@@ -175,7 +175,292 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
                 └── n
 </pre>
 
+Tables {#table-layout}
+----------------------
+The following describes the expected layout for tabular data.
+OME-NGFF tables are compatible with the [AnnData model](https://github.com/scverse/anndata).
 
+<pre>
+.                             # Root folder, potentially in S3,
+│                             # with a flat list of images by image ID.
```

Comment:
I think the "image ID" is a hangover from initial conversion from OMERO, but the name doesn't have to be based on any ID, right?
E.g. "myImage.zarr" instead of "123.zarr" is valid

Reply:
Ah good point. I think this carryover. I will check out how this is specified in `latest` and update.

Reply:
I had a quick look at latest and it looks like the same description there for the Image spec. Is there an PR I should reference to see how this should be updated? In any case, I agree it would be nice to remove the reference to "Image ID".

https://github.com/ome/ngff/blob/642a7a1c6df1afcefa4c4d422920b99681d96a4a/latest/index.bs#L140-L145

Reply:
I think for now, you can just remove the "by image ID" from this line and that will be fine.
Can fix the other "Image ID" in image spec at some other point. Too many changes just now risks conflicts!

Reply:
Makes sense. Done!